tools/ansible/tasks/wsu/main.yaml

```diff
@@ -56,3 +56,24 @@
         url: "https://api-int.{{ cluster_address }}:22623/config/worker"
         dest: "{{ win_temp_dir.path }}\\worker.ign"
         validate_certs: no
+
+    - name: Run bootstrapper
+      win_shell: "{{ win_temp_dir.path }}\\wmcb.exe initialize-kubelet --ignition-file {{ win_temp_dir.path }}\\worker.ign --kubelet-path {{ win_temp_dir.path }}\\kubelet.exe"
+      register: bootstrap_out
+
+    - name: Check if bootstrap was successful
+      fail:
+        msg: "Bootstrapper error"
+      when: '"Bootstrapping completed successfully" not in bootstrap_out.stderr'
+
+    - name: Approve pending bootstrap CSRs
+      delegate_to: localhost
+      shell: "oc get csr | awk '/system:serviceaccount:openshift-machine-config-operator:node-bootstrapper/ && /Pending/ {print $1}' | xargs -r oc adm certificate approve"
```

Comment:
This isn't reflected immediately everytime, maybe add a wait before?

Reply:
added